rotkehlchen/tests/unit/decoders/test_compound_v3.py

```diff
@@ -261,3 +260,247 @@ def test_compound_v3_repay(database, ethereum_inquirer, ethereum_accounts):
+            location_label=polygon_pos_accounts[0],
+            notes=f'Return {return_amount} cUSDCv3 to compound v3',
+            counterparty=CPT_COMPOUND_V3,
+            address=ZERO_ADDRESS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.POLYGON_POS,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=Asset('eip155:137/erc20:0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174'),  # USDC.e,
+            balance=Balance(amount=FVal(withdraw_amount)),
+            location_label=polygon_pos_accounts[0],
+            notes=f'Withdraw {withdraw_amount} USDC from compound v3',
+            counterparty=CPT_COMPOUND_V3,
+            address=string_to_evm_address('0xF25212E676D1F7F89Cd72fFEe66158f541246445'),
+        )]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x6ac5d0D1f2D80f97De90C0b04cc904C257b258Ff']])
+def test_arbitrum_one_borrow(database, arbitrum_one_inquirer, arbitrum_one_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xf1f0e42be8a65d53c903e36a7fbf796108371a6df2f49b858517903fd42fa6df')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1712738344000)
+    gas_fees, borrow_amount = '0.00000136108', '0.367655432709867671'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=arbitrum_one_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=9,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.GENERATE_DEBT,
+            asset=A_ARB,
+            balance=Balance(amount=FVal(borrow_amount)),
+            location_label=arbitrum_one_accounts[0],
+            notes=f'Borrow {borrow_amount} ARB from compound v3',
+            counterparty=CPT_COMPOUND_V3,
+            address=string_to_evm_address('0xA5EDBDD9646f8dFF606d7448e414884C7d905dCA'),
+        )]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0x1397B89726Ada5460C9f995c5EB1a63B6Dc3380B']])
+def test_base_supply(database, base_inquirer, base_accounts):
```

Comment:
shouldn't there be an optimism test too?

Reply:
it's not deployed on optimism. I checked the app and the docs